pkg/api/fs.go

```diff
@@ -101,6 +101,63 @@ func (f *Fs) NewProject(ctx context.Context, req *pb.NewProjectRequest) (*pb.New
 	return &pb.NewProjectResponse{}, nil
 }
 
+func (f *Fs) CloneToProject(ctx context.Context, req *pb.CloneToProjectRequest) (*pb.CloneToProjectResponse, error) {
+	trace.SpanFromContext(ctx).SetAttributes(
+		key.Project.Attribute(req.Source),
+		key.FromVersion.Attribute(&req.FromVersion),
```

Comment:
These are reused from other methods, not 100% sure if that's appropriate

Reply:
The `FromVersion` and `ToVersion` makes sense to be reused, however, I'm not sure we should use `Project` for `req.Source` and `CloneToProject` for `req.Target`. I prefer to match keys to their variables whenever possible.

How do we feel about renaming `source`/`target` to `from_project`/`to_project` and then adding `FromProject`/`ToProject` to the `keys` package?

Reply:
I actually had these originally named `FromProject` to `ToProject` but kept getting them confused with `FromVersion` and `ToVersion`. I'm down to change them back though

Reply:
Oh interesting! I thought using `FromProject`/`ToProject` would be nice because it matches our `ToVersion`/`FromVersion`, but if making them similar causes more confusion than clarity then I'm down to just add `Source` and `Target` to the `keys` package.

I'll let you decide 👍